webextension/confirm-page.html

```diff
@@ -0,0 +1,33 @@
+<html>
+<head>
+  <meta http-equiv="content-type" content="text/html; charset=utf-8">
+  <title>Containers confirm navigation</title>
+  <link xmlns="http://www.w3.org/1999/xhtml" rel="stylesheet" href="chrome://browser/skin/aboutNetError.css" type="text/css" media="all" />
+  <link rel="stylesheet" href="/css/confirm-page.css" />
+</head>
+<body>
+  <main>
+    <div class="title">
+      <h1 class="title-text">Hey, we heard you liked containers!</h1>
+    </div>
+    <form id="redirect-form">
+      <p>
+        A page told us to open:
+      </p>
+      <div id="redirect-url"></div>
+      <p>
+        You asked for <span id="redirect-origin"></span> to always open in <span id="container-name">this</span> container, are you sure you want to confirm?<br />
```

Comment:
Maybe `You assigned <span id="redirect-hostname"></span> to <span id="container-name">this</span> container, ...`